spec/factories/pair_requests.rb

```diff
@@ -26,6 +26,16 @@
     association :author, factory: :user
     association :invitee, factory: :user
     duration { 15.minutes }
+
+    # NOTE: figure out how to override these without using a trait....
```

Comment:
I assume you tried overriding in the job spec, and it didn't work? That seems weird considering how this works in the pair request model spec [here](https://github.com/agency-of-learning/PairApp/blob/main/spec/models/pair_request_spec.rb#L28).